### PR TITLE
feat: added syntax highlights for for expression

### PR DESCRIPTION
**Pull request by okhaimie-dev**
closes: https://github.com/trbutler4/zed-cairo/issues/10

### CODE REVIEW

**Comment by trbutler4** [comment]
we should also update the entire grammar to the most recent commit

File: extension.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@ id = "cairo"
 name = "Cairo"
 version = "0.0.2"
 schema_version = 1
-authors = ["Thomas Robert Butler IV trbiv@proton.me"]
+authors = ["Thomas Robert Butler IV trbiv@proton.me", "Okhai Omotuebe okhaio@protonmail.com"]
 description = "Cairo language support for the Zed editor"
 repository = "https://github.com/trbutler4/zed-cairo"
```
Comment (trbutler4) [inline extension.toml:8]: Instead of just updating the for expression, we should update the entire grammar to the latest commit

**Review by trbutler4 — CHANGES_REQUESTED** [review]

File: extension.toml (hunk 1 above)
Comment (okhaimie-dev) [reply extension.toml:8]: Alright, I'll do this tonight

Comment (okhaimie-dev) [reply extension.toml:8]: @trbutler4 done!

**Review by trbutler4 — APPROVED** [review]
Thank you :)

**Pull request merged**